src/main/java/pl/javastart/task/Ticket.java

```diff
@@ -0,0 +1,109 @@
+package pl.javastart.task;
+
+import java.util.Scanner;
+
+public class Ticket {
+    private String event;
+    private Address address;
+    private String type;
+    private double price;
+    private double discount;
+    private static int id = 1;
```

Comment:
Jeżeli masz tylko pole statyczne, to wszystkie bilety będą miały ten sam id. Potrzebne jest tutaj jeszcze jedno, niestatyczne pole, w którym przechowywałbyś id